### ISSUE TITLE
Can not import data to the emulator

### ISSUE THREAD

**Issue by pascalbe-dev** [jloosli/node-firestore-import-export#859]
When running an import for the local emulator, the data is not really imported to the firestore emulator. 

### Steps to reproduce the behavior

- start the firebase emulator with a demo project ID (`firebase emulators:start --project demo-my-project --config some-folder/firebase.json`) (a firebase.json which includes the activated firestore emulator on port 8080 is present in "some-folder")
-  setting the `FIRESTORE_EMULATOR_HOST` environment variable to `localhost:8080`
- running a import command, e.g.: `firestore-import -y -b users.json -n users`

### Expected behavior

- the data gets imported into the database (this is how it works with actual firebase projects)

### Actual behavior

-  I get an error message about a missing `GOOGLE_APPLICATION_CREDENTIALS` variable

- when I now point this env variable to the service account key for a different environment, the import command works (success message), but data is still not imported in the emulator.

### Final thoughts

So, basically, I see two problems:
- why is the service account key required for the emulator?
- why is the data not imported even with a service account key?

**Comment by cuong0993** [comment]
Forked here https://github.com/cuong0993/node-firestore-import-export

Pass GOOGLE_CLOUD_PROJECT and FIRESTORE_EMULATOR_HOST to connect with emulator.
For example,

GOOGLE_CLOUD_PROJECT=demo- FIRESTORE_EMULATOR_HOST=localhost:8089 firestore-import --backupFile db.json --yes

**Comment by nojaf** [comment]
@cuong0993 I just tried this and it worked like a charm.
Thank you for this!
